pkg/apis/serving/v1/revision_validation.go

```diff
@@ -24,6 +24,7 @@ import (
 	"knative.dev/pkg/kmp"
 	"knative.dev/serving/pkg/apis/autoscaling"
 	"knative.dev/serving/pkg/apis/serving"
+	"knative.dev/serving/pkg/reconciler/route/config"
```

Comment:
The direction of this dependency is a dangerous slippery slope.  Can we move this constant into `serving` alongside the others please?

Can you also fix the other file that this was copied from? (see my other comment)

I'm kind of puzzled that we don't his this in our e2e tests... 🤔

Reply:
sigh - moving it gives me an import cycle for `config/activator.yaml`: 
```
import cycle not allowed
package knative.dev/serving/cmd/activator
        imports knative.dev/serving/pkg/activator/handler
        imports knative.dev/serving/pkg/activator/net
        imports knative.dev/serving/pkg/activator/util
        imports knative.dev/serving/pkg/apis/serving/v1alpha1
        imports knative.dev/serving/pkg/apis/autoscaling/v1alpha1
        imports knative.dev/serving/pkg/apis/serving
        imports knative.dev/serving/pkg/reconciler/route/config
        imports knative.dev/serving/pkg/apis/serving
```
will keep digging